postgres/lib/PostgresService.js

```diff
@@ -212,7 +212,7 @@ GROUP BY k
           if (isBinary) value.setEncoding('base64')
           value.pipe(paramStream)
           value.on('error', err => paramStream.emit('error', err))
-          streams[i] = paramStream
+          streams.push(paramStream)
```

Comment:
could you elaborate why this solves the issue of having `undefined` values in the `streams` array? If `paramStream` is `undefined` wouldn't both arrays contain the `undefined` value?